### DISCUSSION TITLE
Non-admin user cannot see Edit Dashboard button, even on dashboards they own

### DISCUSSION THREAD

**Discussion by ianr21** [apache/superset#24182] [Q&A / Help]
Hi,

On a brand new vanilla superset installation, I did nothing else but created a new user and gave the Gamma role (also tried all other non-admin roles).

This user can create a dashboard, but the `Edit Dashboard` button is missing, so the user cannot add charts. (please see attached image)

**Would appreciate any kind of confirmation if this is likely a problem in my setup, or is that somehow the intended behavior?**

I made sure this user is an owner of that dashboard.
I also tried enabling the DASHBOARD_RBAC feature flag, but same thing.
I also tried duplicating the admin role, still the Edit Dashboard button won't appear. Only if I make that user an Admin, will the button appear.

The user can edit the dashboard's properties though, but just can't add charts.
And the user can see and edit all the charts.

Any advice please? 

![2023-05-22_20-41](https://github.com/apache/superset/assets/31370667/36f50ad4-8260-44bc-9544-4ad8d80e6efc)

Thanks!

**Comment by roks531** [comment]
I am also looking for this functionality but cant seem to find it... Even compared permissions between the Admin and Alpha roles and I cant find anything that would logically identify the permissions if this is a permissions problem.

**Comment by roks531** [comment]
Upgraded to version 2.1.0-dev and the problem is gone, as it seams this must have been a bug which is resolved in later versions. So now when one is the owner, he can also edit the dashboard without being in the Admin role.
